docs/installing_cpython.md

```diff
@@ -122,12 +122,24 @@ and installing the free-threaded binaries is also possible:
     sudo apt-get install python3.13-nogil
     ```
 
-### Conda-forge
+### Conda Packages
+
+=== "Conda-forge"
 
 ```bash
 mamba create -n nogil -c conda-forge/label/python_rc python-freethreading
 ```
 
+=== "Anaconda Testing"
+
+Anaconda's test channel includes the Python interpreter and ABI-compatible builds of many common packages, like NumPy, Cython, Pandas, etc.  These packages use the `python_abi` metapackage and should be compatible with conda-forge:
+
+```bash
+conda create -n nogil -c ad-testing/label/py313 python-freethreading
```

Comment:
When I try to run this command on my ARM Mac, I get:

```
LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides _python_rc needed by python-3.13.0rc1-h17d3ab0_0_cp313t

Could not solve for environment specs
The following packages are incompatible
└─ python-freethreading is not installable because there are no viable options
   ├─ python-freethreading 3.13.0rc2 would require
   │  └─ python 3.13.0.*  but there are no viable options
   │     ├─ python [3.13.0b4|3.13.0rc1|3.13.0rc2] would require
   │     │  └─ tk >=8.6.14,<8.7.0a0 , which conflicts with any installable versions previously reported;
   │     └─ python [3.13.0rc1|3.13.0rc2] would require
   │        └─ _python_rc, which does not exist (perhaps a missing channel);
   └─ python-freethreading [3.13.0rc1|3.13.0rc2] conflicts with any installable versions previously reported.
```

Any idea what's causing that?

Reply:
Thanks for taking a look Nathan!

Are you running that exact command? It seems to work fine on my ARM mac:
`conda create -n nogil -c ad-testing/label/py313 python-freethreading`

conda --version
conda 24.7.1

Otherwise, I'm wondering if you're trying to add this package into an existing environment, leading to a conflict?

Reply:
tk 8.6.14 is provided by the defaults repository for my install:
Channels:
 - ad-testing/label/py313
 - defaults
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: done

  added / updated specs:
    - python-freethreading


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    expat-2.6.3                |       h313beb8_0         154 KB
    libmpdec-4.0.0             |       h80987f9_0          69 KB
    openssl-3.0.15             |       h80987f9_0         4.3 MB
    pip-24.2                   |  py313hf88f6fa_0         2.7 MB  ad-testing/label/py313
    python-3.13.0rc2           |haa6bb3f_0_cp313t        15.5 MB  ad-testing/label/py313
    python-freethreading-3.13.0rc2|  py313h56471e5_0          52 KB  ad-testing/label/py313
    python_abi-3.13            |         5_cp313t           5 KB  ad-testing/label/py313
    setuptools-72.1.0          |  py313hf88f6fa_0         3.1 MB  ad-testing/label/py313
    wheel-0.43.0               |  py313hf88f6fa_0         132 KB  ad-testing/label/py313
    ------------------------------------------------------------
                                           Total:        26.1 MB

The following NEW packages will be INSTALLED:

  bzip2              repo/main/osx-arm64::bzip2-1.0.8-h80987f9_6
  ca-certificates    repo/main/osx-arm64::ca-certificates-2024.7.2-hca03da5_0
  expat              repo/main/osx-arm64::expat-2.6.3-h313beb8_0
  libcxx             repo/main/osx-arm64::libcxx-14.0.6-h848a8c0_0
  libffi             repo/main/osx-arm64::libffi-3.4.4-hca03da5_1
  libmpdec           repo/main/osx-arm64::libmpdec-4.0.0-h80987f9_0
  ncurses            repo/main/osx-arm64::ncurses-6.4-h313beb8_0
  openssl            repo/main/osx-arm64::openssl-3.0.15-h80987f9_0
  pip                ad-testing/label/py313/osx-arm64::pip-24.2-py313hf88f6fa_0
  python             ad-testing/label/py313/osx-arm64::python-3.13.0rc2-haa6bb3f_0_cp313t
  python-freethread~ ad-testing/label/py313/noarch::python-freethreading-3.13.0rc2-py313h56471e5_0
  python_abi         ad-testing/label/py313/osx-arm64::python_abi-3.13-5_cp313t
  readline           repo/main/osx-arm64::readline-8.2-h1a28f6b_0
  setuptools         ad-testing/label/py313/osx-arm64::setuptools-72.1.0-py313hf88f6fa_0
  sqlite             repo/main/osx-arm64::sqlite-3.45.3-h80987f9_0
  tk                 repo/main/osx-arm64::tk-8.6.14-h6ba3021_0
  tzdata             repo/main/noarch::tzdata-2024a-h04d1e81_0
  wheel              ad-testing/label/py313/osx-arm64::wheel-0.43.0-py313hf88f6fa_0
  xz                 repo/main/osx-arm64::xz-5.4.6-h80987f9_1
  zlib               repo/main/osx-arm64::zlib-1.2.13-h18a0788_1


Proceed ([y]/n)? y

Reply:
Sorry for the multiple posts, it looks like this issue happens when you have `conda-forge` as a channel but not defaults.

Conda forge doesn't have a compatible version of tk.

```
conda create -p ./testnogil -c ad-testing/label/py313 -c conda-forge python-freethreading
Channels:
 - ad-testing/label/py313
 - conda-forge
 - anaconda-cloud
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides tk >=8.6.14,<8.7.0a0 needed by python-3.13.0b4-h9facb3d_0_cpython_cp313

Could not solve for environment specs
The following package could not be installed
└─ python-freethreading is not installable because it requires
   ├─ cpython 3.13.0.* , which requires
   │  └─ python 3.13.0.*  but there are no viable options
   │     ├─ python [3.13.0b4|3.13.0rc1|3.13.0rc2] would require
   │     │  └─ tk >=8.6.14,<8.7.0a0 , which does not exist (perhaps a missing channel);
   │     └─ python [3.13.0rc1|3.13.0rc2] would require
   │        └─ _python_rc, which does not exist (perhaps a missing channel);
   └─ python 3.13.0.* , which cannot be installed (as previously explained).
```

https://anaconda.org/conda-forge/tk/files
Their latest version is 8.6.13

Reply:
> it looks like this issue happens when you have conda-forge as a channel but not defaults

Yup, that's it, conda and mamba both are configured to search conda-forge by default on my machine. I don't think that's uncommon either.

Reply:
Perhaps adding `--override-channels` as well as the defaults/main channel to the command would be more complete.  A complete command would be `conda create -n nogil --override-channels -c ad-testing/label/py313 -c defaults python-freethreading` or if defaults is set in .condarc, `conda create -n nogil --override-channels -c ad-testing/label/py313 -c https://repo.anaconda.com/pkgs/main python-freethreading`

Reply:
Nice! Thanks for the suggestion, I just wanted to have a command that didn't have known issues associated with it. It sounds like the second option using `-c https://repo.anaconda.com/pkgs/main python-freethreading` works on my setup with both conda and mamba.